### DISCUSSION TITLE
[DataObject] WYSIWYG links only showing via GraphQL when data component is wrapped inside block

### DISCUSSION THREAD

**Discussion by RandomIAm** [pimcore/pimcore#10121] [Q&A / Support]
While accessing different DataObjects containing a WYSIWYG data component we ran into the issue of links being dropped during GraphQL requests when those WYSIWYG components themselves where not wrapped inside a Repeater/Block element.

Example: I have a `TextColums` DataObject where my WYSIWYG data component `text` lies within the block `contentlist`. Within my texts I want to drag&drop one or more additional DataObjects in order to link to them. Now, I can easily build a GraphQL query in order to fetch my list texts:

```
{
  "data": {
    "getTextColumns": {
      "id": "376",
      "contentlist": [
        {
          "text": "<p><a data-tabindex-counter=\"1\" data-tabindex-value=\"none\" href=\"/someurl.com/Home\" pimcore_id=\"3\" pimcore_type=\"object\" tabindex=\"-1\">Home</a>&nbsp;Inspiration is an unconscious burst of creativity in a literary, musical, or visual art and other artistic endeavours. The concept has origins in both Hellenism and Hebraism. The Greeks believed that inspiration or \"enthusiasm\" came from the muses, as well as the gods Apollo and Dionysus.</p>\n"
        },
        [...more texts here...]
      ]
    }
  }
}
```

However, even if I was to copy the contents of the above `text` field to one of my other DataObjects where my WYSIWYG component is not wrapped inside a Block, the GraphQL response would simply omit the link (Display Text still shows):

```
{
  "data": {
    "getProductAccessTeaser": {
      "id": "72",
      "text": "<p>Home&nbsp;Inspiration is an unconscious burst of creativity in a literary, musical, or visual art and other artistic endeavours. The concept has origins in both Hellenism and Hebraism. The Greeks believed that inspiration or \"enthusiasm\" came from the muses, as well as the gods Apollo and Dionysus.</p>\n"
    }
  }
}
```

**Comment by weisswurstkanone** [comment]
Hello @RandomIAm Not sure how copying content is anyhow GraphQL related. I woill move this to a discussion now. Maybe you find some time to investigate this further.
